spot_examples/spot_examples/send_inverse_kinematics_requests.py

```diff
@@ -37,13 +38,28 @@
 from bosdyn.client.robot_command import RobotCommandBuilder
 from bosdyn_msgs.conversions import convert
 from rclpy.node import Node
-from tf2_ros import TransformBroadcaster
-from utilities.simple_spot_commander import SimpleSpotCommander
-from utilities.tf_listener_wrapper import TFListenerWrapper
+from tf2_ros import TransformBroadcaster, TransformStamped
 
 from spot_msgs.action import RobotCommand  # type: ignore
 from spot_msgs.srv import GetInverseKinematicSolutions  # type: ignore
 
+from .simple_spot_commander import SimpleSpotCommander
+
+
+def to_se3(ros_transform: TransformStamped) -> SE3Pose:
+    """Convert from ROS TransformStamped to Bosdyn SE3Pose"""
+    return SE3Pose(
+        ros_transform.transform.translation.x,
+        ros_transform.transform.translation.y,
+        ros_transform.transform.translation.z,
+        Quat(
+            ros_transform.transform.rotation.w,
+            ros_transform.transform.rotation.x,
+            ros_transform.transform.rotation.y,
+            ros_transform.transform.rotation.z,
+        ),
+    )
+
 
 class SpotRunner:
```

Comment:
Can we rework this function use the `ros_process`? If we're already forcing a dependency on `bdaii_ros2_wrappers`, I would rather have it depend entirely on the `ros_process` instead of the standard `ROS2` stuff. Either that or have one example using the scope-aware process and one using vanilla ROS2.

Reply:
Yeah, it would be nice to have an example in this repo that uses `ros_process`. I can look into it in a future PR since the focus here was just reorganization